oem/ibm/libpldmresponder/oem_ibm_handler.cpp

```diff
@@ -1019,20 +1140,32 @@ int pldm::responder::oem_ibm_platform::Handler::setNumericEffecter(
                                           interface, "CreateDump");
 
         std::map<std::string, std::variant<std::string, uint64_t>> createParams;
-        createParams["com.ibm.Dump.Create.CreateParameters.DumpType"] =
-            "com.ibm.Dump.Create.DumpType.SBE";
-        createParams["com.ibm.Dump.Create.CreateParameters.ErrorLogId"] =
-            (uint64_t)value;
-        createParams["com.ibm.Dump.Create.CreateParameters.FailingUnitId"] =
-            (uint64_t)entityInstance;
+        if (entityType == PLDM_ENTITY_MEMORY_MODULE)
+        {
+            createParams["com.ibm.Dump.Create.CreateParameters.DumpType"] =
```

Comment:
Create static constexpr auto for "com.ibm.Dump.Create.CreateParameters.DumpType"

Reply:
Done

Reply:
Done